docs/business.yaml

```diff
@@ -1564,15 +1564,604 @@ paths:
+                          deliveryAddress: 
+                            addressCity: Victoria
+                            addressCountry: CA
+                            addressRegion: BC
+                            deliveryInstructions: 
+                            postalCode: V8Z 5C5
+                            streetAddress: Test rd
+                            streetAddressAdditional:
+                          mailingAddress: 
+                            addressCity: Victoria
+                            addressCountry: CA
+                            addressRegion: BC
+                            deliveryInstructions: 
+                            postalCode: V8Z 5C5
+                            streetAddress: Test rd
+                            streetAddressAdditional:
+                        registeredOffice: 
+                          deliveryAddress: 
+                            addressCity: Victoria
+                            addressCountry: CA
+                            addressRegion: BC
+                            deliveryInstructions: 
+                            postalCode: V8Z 5C5
+                            streetAddress: Test rd
+                            streetAddressAdditional: 
+                          mailingAddress: 
+                            addressCity: Victoria
+                            addressCountry: CA
+                            addressRegion: BC
+                            deliveryInstructions: 
+                            postalCode: V8Z 5C5
+                            streetAddress: Test rd
+                            streetAddressAdditional:
+                      parties:
+                        - mailingAddress:
+                            addressCity: Victoria
+                            addressCountry: Canada
+                            addressCountryDescription: Canada
+                            addressRegion: BC
+                            deliveryInstructions: ''
+                            postalCode: V8Z 5C5
+                            streetAddress: 3950 Helen Rd
+                            streetAddressAdditional: ''
+                          officer:
+                            email: null
+                            firstName: Completing
+                            lastName: Party
+                            partyType: person
+                          roles:
+                            - appointmentDate: '2024-07-24'
+                              roleType: Completing Party
+                        - mailingAddress:
+                            addressCity: Victoria
+                            addressCountry: Canada
+                            addressCountryDescription: Canada
+                            addressRegion: BC
+                            deliveryInstructions: ''
+                            postalCode: V8Z 5C5
+                            streetAddress: 3950 Helen Rd
+                            streetAddressAdditional: ''
```

Comment:
Based on the request body in the Postman collection. Seems like missing a `deliveryAddress` after or before the `mailingAddress`.